### DISCUSSION TITLE
Who uses BEM

### DISCUSSION THREAD

**Discussion by floatdrop** [getbem/getbem.github.io#282] [Q&A]
If you or your company using BEM methodology, we will be happy add you logotype at [getbem.com](http://getbem.com/#companies).

If you can add link to repository with build scripts or write a quote about `How BEM make your life better` - we will pick best ones and put them on site too.

**Comment by floatdrop** [comment]
@varya @ingdir could we add your company logos?

**Reply by SunnatAxmatovWebdev** [reply]
I don't know

**Comment by iamstarkov** [comment]
@voischev can we add http://factory.mn/ logo too?

**Comment by voischev** [comment]
@matmuchrapna yes. what you need from us?

**Comment by floatdrop** [comment]
@voischev logotype. little story about your BEM experience would be great!

**Comment by voischev** [comment]
@floatdrop how many words of text?

**Comment by floatdrop** [comment]
@voischev it is optional. We are planning to post quotes about BEM on site, so try to split it into two parts - one about a twit size, and (if you want) second, that not bounded by size.

**Comment by voischev** [comment]
http://factory.mn 
logo https://yadi.sk/d/E0USvLdXbToKg

«Manufactura» is a company that creates websites and services with beautiful design. Full stack of BEM technology helps our development team implement any highly creative ideas fast and qualitatively.

**Comment by iamstarkov** [comment]
@voischev does Manufactura have international site?

**Comment by iamstarkov** [comment]
Thanks @delka we can add http://eu.ideus.biz/ to this list :trophy:

**Comment by iamstarkov** [comment]
First iteration with Yandex, Manufactura and IDeus in the list landed in 27c5bd4dca8a1a7cb6111709a730269f4e3e2a64

**Comment by voischev** [comment]
@matmuchrapna no

**Comment by awinogradov** [comment]
@matmuchrapna Alfa-Bank uses BEM too

**Comment by floatdrop** [comment]
@verybigman great! Could you provide link to png logo which should we put on site?

**Comment by awinogradov** [comment]
@floatdrop [this](http://alfabank.ru/about/ads/logo/) is official

**Comment by h4** [comment]
And we also use BEM, but not full-stack. https://edster.ru/

**Comment by floatdrop** [comment]
@h4 awesome! could you point to png logotype?

**Comment by h4** [comment]
![page__logo](https://cloud.githubusercontent.com/assets/430023/4697588/56da5d5a-5829-11e4-9d96-589da416e088.png)

It's white letters on transparent background, therefore it looks like bug :)

**Comment by floatdrop** [comment]
@h4 got ya - I inverted it (because our background is also white), I hope it's fine.

`Alfa Bank` and `edster` landed in https://github.com/getbem/getbem.com/commit/78e25bd8a428cbb4a926c56664f76fccd7f301d2 and https://github.com/getbem/getbem.com/commit/bf0f3b545ea7d6aa631eab8630526401fa1e148a

**Comment by h4** [comment]
@floatdrop great! Thank you!

**Comment by floatdrop** [comment]
I wonder if @yandex is ok with using logo.

**Comment by iamstarkov** [comment]
I think it is, because this logo was taken from official logo page http://company.yandex.com/general_info/logotype.xml

**Comment by qfox** [comment]
Pretty sure that MegaFon also uses BEM. You can see it on megafon.ru and lk.megafon.ru

**Comment by iamstarkov** [comment]
@zxqfox do you know any developer from megafon who can and want to give us megafon’s logo?

**Comment by qfox** [comment]
@matmuchrapna I have this:

http://moscow.megafon.ru/b-header/i/logo-big.png
![image](https://cloud.githubusercontent.com/assets/677518/7163840/d13bf2aa-e3a4-11e4-8e36-df03c024b4e6.png)

**Comment by qfox** [comment]
http://moscow.megafon.ru/b-header/i/logo2x.png — a little bit larger
http://moscow.megafon.ru/b-header/i/logo.svg — svg version

I'm trying to find english.

**Comment by iamstarkov** [comment]
I vote contra. `b-options__itemLink__icon__iconChar b-options__itemLink__icon__iconChar_rocket` seems like broken bem.
![screen shot 2015-04-15 at 20 47 20](https://cloud.githubusercontent.com/assets/559321/7166823/a3f8b1b4-e3b0-11e4-8936-6c6c101722ca.png)

**Comment by qfox** [comment]
Yup, that's what we refactoring now. Index page and common layouts contains strange names. But in common it uses BEM (even `i-bem`).

**Comment by iamstarkov** [comment]
oh-oh, so you are that one developer from megafon? It’s cool. why din’t you tell it earlier? btw, your github profile is telling about "Dalee IA" =) 

anyway, if you are refactoring now, then it seems to be ok, to include megafon in this list

**Comment by qfox** [comment]
;-D I'm not sure why I should. Natural modesty rocks!

btw. http://failoverconf.ru/upload/iblock/133/2_04_Kobychenko.pdf — we had a talk about that refactoring at failover conf. On slide 14 we crying about that.

**Comment by iamstarkov** [comment]
@zxqfox you should contact your PR’s dept, cause english site is kind of broken http://english.corp.megafon.ru/news/gallery/ and also have no brand assets page =( e.g. twitter have one in proper way https://about.twitter.com/press/brand-assets

**Comment by iamstarkov** [comment]
I found english version ![png image 143 x 35 pixels](https://cloud.githubusercontent.com/assets/559321/7169858/8a94990e-e3c9-11e4-9c4b-6cf793565f74.png) Is this version acceptable? what do you think? @zxqfox @floatdrop @varya @ingdir

**Comment by qfox** [comment]
Our PO's said me that rework of these sites in a timeline. I hope we will fix them soon ;-)

Yeah, it's an actual version, but it's so small. ~~I've asked for svg.~~

svg: https://yadi.sk/d/ATmSdOCVg4ixM
png: https://yadi.sk/i/9w6Nn5FDg4j26

**Comment by iamstarkov** [comment]
@zxqfox  is there any news?

**Comment by qfox** [comment]
@vladimirstarkov English svg, and big png attached. Thanks :smirk_cat:

**Comment by qfox** [comment]
![image](https://cloud.githubusercontent.com/assets/677518/7244481/e456879e-e7e4-11e4-9842-79977e54790c.png)

**Comment by uyeong** [comment]
![2015-10-29 1 53 18](https://cloud.githubusercontent.com/assets/1321707/10810470/b3795ba0-7e44-11e5-94b8-0315c5ffca80.png)
"SOUNDCLOUD" also seems like BEM.

**Comment by ihorzenich** [comment]
EPAM uses BEM too!
Not full-stack, like many other.
Now I can't provide examples of code due to NDA, maybe I can do it later…
- Our SVG Logo: http://www.epam.com/content/dam/epam/global/logo_gray-blue_80px.svg
- Company URL: http://epam.com

**Comment by iamstarkov** [comment]
> Now I can't provide examples of code due to NDA, maybe I can do it later…

if you are using it then, you do it in prod?

**Comment by ihorzenich** [comment]
@iamstarkov Yes, in production, on different projects, for different clients, worldwide, I saw projects from EPAM USA, and they use BEM too.
I will ask Head of UX Unit Kharkiv to provide us approved links a projects builded by EPAM using BEM. It will take some time :)

Also, please use this version of EPAM logo, for putting it on solidcolor background (like other logos on getbem): https://upsa-880739.c.cdn77.org/workload/images/epam-logo.svg

**Comment by iamstarkov** [comment]
like this? 
![screen shot 2015-11-13 at 15 08 28](https://cloud.githubusercontent.com/assets/559321/11147745/6e1449f0-8a18-11e5-9170-9e8138d2e07e.png)

**Comment by ihorzenich** [comment]
@iamstarkov I try to follow our guidelines: http://take.ms/BZ5hb but it seems like first type of logo (gray-blue) will be better. Can you try?

**Comment by ihorzenich** [comment]
@iamstarkov I will ask my manager for logo colors, and then prepare Pull Request, ok? As for order — I should add „new company used BEM“ to the end of list, right?

**Comment by iamstarkov** [comment]
right. looking forward for you pr 💫

**Comment by ihorzenich** [comment]
Btw, Google recommends BEM in his Guidelines and create MDL framework based on BEM, so I think we should add Google to “Companies used BEM”.

**Comment by azizur** [comment]
BBC (Homepage, Sport, and News) uses BEM.

**Comment by iamstarkov** [comment]
@delka is there any news about pr?

**Comment by iamstarkov** [comment]
@azizur great news! would you like to add BBC logo to the list?

**Comment by ihorzenich** [comment]
@iamstarkov My managers says me, what I should get approve from marketing department to put EPAM logo somewhere. So I in progress with it from Friday and try to do my best))
Btw, I found out, what EPAM has experience with Full Stack BEM too, so we can confidently say „EPAM able to do BEM“.

**Comment by iamstarkov** [comment]
@delka can you ping them =) its already tuesday

**Comment by ihorzenich** [comment]
@iamstarkov I should get approve from Dev department first, and already discuss with them.

**Comment by gerhard-boden** [comment]
Hi Guys,

we at gesagt.getan. (web agency based in austria) really enjoy using BEM! We use it for all our new projects.
![gg_logo_4c](https://cloud.githubusercontent.com/assets/10533739/11240322/0a322ffa-8df2-11e5-984c-56070f3b75f9.jpg)

Website: https://www.gesagt-getan.at
(we haven't gotten around to update our own website to BEM)

**Comment by ihorzenich** [comment]
Colleagues from JetBrains, Manufactura, Alfa-Bank, Эдстер, WIMDU, Megafon, DECAF, could you provide links to the sites using BEM, written in the company where you work?
I want it for my new presentation „BEM in the Wild“.

**Comment by nxame** [comment]
I have started advocating and using it here in Microsoft India for our customer projects, though I have been using it personally since long.

**Comment by nxame** [comment]
Unfortunately I can't provide any public site links yet reason being we mainly develop business enterprise applications and are not available for open public use. If I ever do so - I will post a link here.

**Comment by ihorzenich** [comment]
And finally I received approval to add EPAM logo to „Who uses BEM“ section!
See my PR #82 please.

**Comment by ooaeoaii** [comment]
http://evo.company/
logo: http://i.imgur.com/IgKq93T.png

**Comment by alexspark86** [comment]
Please update JetBrains logo: https://resources.jetbrains.com/assets/jetbrains/jetbrains_logos.zip

**Comment by jgillman** [comment]
![goldbely-logo-optimized](https://cloud.githubusercontent.com/assets/172217/19748065/6cbd2e58-9b94-11e6-8756-6a3ea2d031c3.png) uses BEM! 

Our repo is private, but you can see it in action on the site: https://www.goldbely.com

In particular it's made our large CSS less cryptic, more predictable, and easier to maintain. It's also forced us (for the better) to think of our CSS in more modular and reusable ways. IDK where I'd be without BEM.

**Comment by tzi** [comment]
One of the last Google website ([Made by Google](https://madeby.google.com/phone/)) use BEM.
Here is a link to the [Google Logo](https://lh3.googleusercontent.com/-ZxK7RcVQdOE/Vd5bJfv4tYI/AAAAAAAAQ70/p8tAh6KU-OQfaAEiXteXDCpzLXe1jag6gCHM/d/logo_420_color_2x.png) from the [press page](https://www.google.com/press/images.html).

**Comment by mazipan** [comment]
@floatdrop 

[Blibli.com](https://www.blibli.com/) - a Big E-commerce in Indonesia has been created one site called [Blibli.com Travel](https://www.blibli.com/travel/) that using BEM Convention.
You can check website [here](https://www.blibli.com/travel/).
Here the logo : 
+ Link : [Blibli.com Travel](https://www.static-src.com/travel/2.1.1/resources/hotel/images/thumbnail-blibli-travel.png)
+ Preview : ![Blibli.com Travel Preview](https://www.static-src.com/travel/2.1.1/resources/hotel/images/thumbnail-blibli-travel.png)

So far, we very enjoy using BEM while we still learning to implement BEM better.
We remove many unused CSS and have no afraid to update and do any changes in any CSS style with BEM convention.

Thank you so much :+1:

**Comment by mahmadinurov** [comment]
Could I ask you please post our logo on BEM web-site.

Link: http://knopka.com/
Logo: https://s21.postimg.org/lcs66dt5z/KNOPKA_logo_ENG_RGB_500px.png

Thanks a lot.

**Comment by pjhampton** [comment]
[DuckDuckGo](https://duckduckgo.com/) uses BEM.
You can see it in action in their open source ZeroClickInfo repos: https://github.com/duckduckgo

**Comment by manuel-minniti** [comment]
I guess Dropbox would be a prestigious company to advertise.

https://github.com/dropbox/css-style-guide#bem

**Comment by VienDinhCom** [comment]
Please add Google, they use BEM for `https://getmdl.io` and `https://material.io/develop/web/`

**Comment by matdec01** [comment]
Please add Arm (http://www.arm.com)

**Comment by jstblck** [comment]
The UK government's GOV.UK uses BEM ([see here](https://design-system.service.gov.uk/get-started/updating-your-code/#class-names)).

**Comment by shaundon** [comment]
We use BEM at [Skyscanner](https://skyscanner.net), both in our [design system Backpack](https://backpack.github.io) and internally.

**Comment by chrysillala** [comment]
[Gojek website](https://www.gojek.com) uses BEM, along with Dropbox namespaced classes

**Comment by raghavdhingra-bst** [comment]
We at @bluestacks use BEM in our Web Applications

**Comment by sergei201613** [comment]
Unity Technologies using BEM in their new (UITookit / UIElements) user interface system.

**Comment by chalkedgoose** [comment]
linkedin.com uses BEM a lot

**Comment by MIRZOULUGBEK9008** [comment]
I always use the BEM in my all projects

**Comment by banderon** [comment]
We've used BEM at [TechCrunch](https://techcrunch.com) for more than half a decade.

**Comment by Rusya13** [comment]
We use BEM at [School Holidays](https://schoolholidays.com.au).

**Comment by madhusudanbabar** [comment]
I'm using BEM since last 3 years at [madhusudan.live](https://madhusudan.live)

**Comment by xmadscientist** [comment]
We use BEM at [Olsa Tools](https://olsatools.com)

**Comment by felipeRAMOS2612** [comment]
Hello, i use BEM in [ClubRv](https://cl.casinoclubrv.com)

**Comment by dev-bazz** [comment]
Hi I use BEM
